runtime/doc/deprecated.txt

```diff
@@ -23,8 +23,8 @@ LUA
 - vim.region()		Use |getregionpos()| instead.
 
 DIAGNOSTICS
-- *vim.diagnostic.goto_next()*	Use |vim.diagnostic.jump()| with `{count = 1}` instead.
-- *vim.diagnostic.goto_prev()*	Use |vim.diagnostic.jump()| with `{count = -1}` instead.
+- *vim.diagnostic.goto_next()*	Use |vim.diagnostic.jump()| with `{count = 1, float=true}` instead.
+- *vim.diagnostic.goto_prev()*	Use |vim.diagnostic.jump()| with `{count = -1, float=true}` instead.
```

Comment:
Nit, use spaces around `=` in both or in neither, but don't mix (since we're space constrained, probably omit spaces):

```suggestion
- *vim.diagnostic.goto_next()*	Use |vim.diagnostic.jump()| with `{count=1, float=true}` instead.
- *vim.diagnostic.goto_prev()*	Use |vim.diagnostic.jump()| with `{count=-1, float=true}` instead.
```